lizmap/modules/lizmap/lib/Project/ProjectMetadata.php

```diff
@@ -230,6 +231,23 @@ public function needsUpdateWarning()
         return $this->data['needsUpdateWarning'];
     }
 
+    /**
+     * Returns the count of warnings in the CFG file.
+     *
+     * @return int
+     */
+    public function countProjectCfgWarnings()
+    {
+        // Before plugin 4.0.0, it was a list of errors
```

Comment:
any example , i don't get it ?

array or list , count will be the same 

also no need to cast type

Reply:
Thanks,
PR updated, I think it's better now.

Cast is needed, otherwise, we have a "[array_is_list(): Argument #1 ($array) must be of type array, stdClass given](http://lizmap.local:8130/admin.php/admin/qgis_projects#)"